R/support.R

```diff
@@ -637,6 +641,53 @@ compute_pathway_probabilities <- function(step, pars, n_time_steps, n_strains,
   out
 }
 
+##' Checks severity probabilities are valid
+##'
+##' @title Check severity parameters
+##'
+##' @param pars A list of probabilities from [carehomes_parameters].
+##'
+##' @return Returns `pars` invisibly if all severity probabilities lie in
+##   (0, 1), otherwise errors.
+##'
+##' @export
+carehomes_check_severity <- function(pars) {
```

Comment:
minor suggestion: since this is carehomes model specific (and features carehomes in the function name), I think this should be moved into carehomes.R (and the corresponding tests moved to test-carehomes-support.R)